src/Builder.php

```diff
@@ -93,7 +93,7 @@ protected function getEndpoint()
     {
         $result = $this->getTarget();
 
-        if (! empty($this->getResource())) {
+        if (! $this->getResource()) {
```

Comment:
It causes the testing to fail as it completely inverts the logic of the condition. I think it can be left as:

```php
if ($this->getResource()) {
```